src/test/java/net/minecraftforge/debug/world/PotentialSpawnsEventTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.world;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.Difficulty;
+import net.minecraft.world.entity.MobCategory;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.world.WorldEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("potentialspawnsevent_test")
+public class PotentialSpawnsEventTest
+{
+    public static final boolean enabled = false;
+
+    public PotentialSpawnsEventTest()
+    {
+        if (enabled)
+        {
+            MinecraftForge.EVENT_BUS.addListener(PotentialSpawnsEventTest::onlySpawnHostileMobs);
+        }
+    }
+
+    public static void onlySpawnHostileMobs (WorldEvent.PotentialSpawns event)
+    {
+        Level level = (Level) event.getWorld();
```

Comment:
Why is this being casted to `Level`, given that the only use of the value is to call `LevelAccessor#getCurrentDifficultyAt`?

Reply:
Good catch, a holdover from when I was playing with different ideas for the test. Fixed locally.